grass7/.flake8

```diff
@@ -0,0 +1,50 @@
+[flake8]
+ignore =
+    E121, # continuation line under-indented for hanging indent
+    E125, # continuation line with same indent as next logical line
+    E127, # continuation line over-indented for visual indent
+    E128, # continuation line under-indented for visual indent
+    E202, # whitespace before ')'
+    E211, # whitespace before '['
+    E221, # multiple spaces before operator
+    E225, # missing whitespace around operator
+    E226, # missing whitespace around arithmetic operator
+    E231, # missing whitespace after ','
+    E251, # unexpected spaces around keyword / parameter equals
+    E261, # at least two spaces before inline comment
+    E265, # block comment should start with '# '
+    E266, # too many leading '#' for block comment
+    E271, # multiple spaces after keyword
+    E272, # multiple spaces before keyword
+    E302, # expected 2 blank lines, found 1
+    E303, # too many blank lines (2)
+    E305, # expected 2 blank lines after class or function definition, found 1
+    E402, # module level import not at top of file
+    E501, # line too long (161 > 150 characters)
+    E722, # do not use bare 'except'
+    E741, # ambiguous variable name 'l'
+    F401, # 'grass.script.core.gisenv' imported but unused
+    F403, # 'from ... import *' used; unable to detect undefined names
+    F405, # '_' may be undefined, or defined from star imports: datetime
+    F821, # undefined name '_'
+    F841, # local variable 'center' is assigned to but never used
+    W293, # blank line contains whitespace
+    W503, # line break before binary operator
+    W504, # line break after binary operator
+    W605, # invalid escape sequence '\_'
+
+
+max-line-length = 88
+exclude =
+    .git,
+    __pycache__,
+    .env,
+    .venv,
+    env,
+    venv,
+    ENV,
+    env.bak,
+    venv.bak,
+    ctypes,
+    pydispatch,
```

Comment:
No need to exclude ctypes and pydispatch. There are no files like this. These are in core, in `lib/python`. (Same of the other flake file.)